registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/utils/CheckSumGeneration.java

```diff
@@ -63,15 +67,15 @@ public CheckSumGeneration(PacketManager adapter) {
 
 			if (PacketFiles.APPLICANTBIOMETRICSEQUENCE.name().equalsIgnoreCase(fieldValueArray.getLabel())) {
 
-				generateBiometricsHash(fieldValueArray.getValue(), registrationId, outputStream);
+				generateHash(fieldValueArray.getValue(), registrationId, outputStream);

```

Comment:
"hashSequence1" : [ {
    "label" : "demographicSequence",
    "value" : [ "proofOfIdentity", "ID.json" ]
  } ],
  "hashSequence2" : [ {
    "label" : "otherFiles",
    "value" : [ "audit.json" ]
  } ],
This is finalised hashsequence,Please change the code according  to that